README.md

```diff
@@ -2,3 +2,17 @@
 
 We use this repository as a forum to discuss and record technical decisions that
 affect the products supported by the GOV.UK Design System team, including:
+
+- GOV.UK Design System
+- GOV.UK Frontend
+- GOV.UK Prototype Kit
+
+This repository is open to enable involvement from the wider community and as a
+reference for other teams within GDS and wider government.
+
+There are 2 types of documentation:
+
+- [Proposals](/proposals), often referred to as 'requests for comments', which
+  allow us and the wider community to suggest and refine ideas.
+- [Decision Records](/decision-records), which document decisions that have been
```

Comment:
Opinion (2/10): Could this be just 'decisions'?

Reply:
It could, but I think it _might_ make it harder to talk about as a noun? "Include any images etc in a separate directory with the same name as your decision", "The decision will be reviewed.", etc. It makes it hard to distinguish between the actual 'decision' and 'the thing that's being reviewed'.